src/workerd/api/global-scope.c++

```diff
@@ -706,7 +706,20 @@ void ServiceWorkerGlobalScope::queueMicrotask(jsg::Lock& js, v8::Local<v8::Funct
               return jsg::check(
                   function->Call(js.v8Context(), js.v8Null(), argv.size(), argv.data()));
             }, [&](jsg::Value exception) {
-              reportError(js, jsg::JsValue(exception.getHandle(js)));
+              // The reportError call itself can potentially throw errors. Let's catch
+              // and report them as well.
+              js.tryCatch([&] { reportError(js, jsg::JsValue(exception.getHandle(js))); },
+                  [&](jsg::Value exception) {
+                if (IoContext::hasCurrent()) {
+                // If we're running in a request context then we want to abort the
```

Comment:
BTW what's with the indentation here? Seems all wrong but the formatter isn't complaining?

Reply:
cc @danlapid @npaun any ideas?

Reply:
Clang format was disabled by https://github.com/cloudflare/workerd/pull/2785
Until https://github.com/cloudflare/workerd/pull/2791 gets merged.

Reply:
Doesn't CI still enforce?

Reply:
Nope, sadly the above mentioned PR just disabled it completely both for local formatting and for CI

Reply:
OK, but still, the formatter was disabled _after_ this PR was uploaded... so this PR did pass the formatter before it was disabled, seemingly.

Reply:
Yeah, the formatting here was picked by the formatter, not by anything I did. I updated and checked in with `--no-verify` to ensure good formatting on the updated commit.